src/processes/imageClassification.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+import fs from 'fs';
+import { ItemValidationStatuses, PREDICTION_THRESHOLD } from '../constants';
+import { FaildImageClassificationRequestError } from '../errors';
+import { classifierEndpoint } from '../plugin';
```

Comment:
The endpoint cannot be imported from the plugin like that, I think you have to pass it via the function's parameters.